### ISSUE TITLE
New endpoints to /delete /private a /restore an HLS file

### ISSUE THREAD

**Issue by rushmore-ops** [Rushmorefm/hls-proxy#5]
`/private/{broadcast_id}` will make the hls endpoint point to a placeholder private video
`/delete/{broadcast_id}` will make the hls endpoint point to a placeholder delete video
`/restore/{broadcast_id}` will put it back

not playable (hidden) broadcasts can be 'frozen' into glacier storage.

**Comment by jeoliva** [comment]
With the new approach this depends on #15. I am going to keep this one on hold, migrate all the file operations to work with S3 API and then will finish this one.

**Comment by jeoliva** [comment]
Done
